FluidNC/src/Spindles/HuanyangSpindle.cpp

```diff
@@ -222,8 +223,9 @@ namespace Spindles {
             return [](const uint8_t* response, Spindles::VFD* vfd) -> bool {
                 uint16_t value = (response[4] << 8) | response[5];
 #ifdef DEBUG_VFD
-                log_debug("VFD: Max frequency = " << value);
+                log_debug("VFD: Max frequency = " << value / 100 << "Hz " << value / 100 * 60 << "RPM");
```

Comment:
Careful, this is integer math. `value/100*60` does `(value/100)*60`, which is probably incorrect. It should be `(value*60)/100`.

I also make it a habit to put the math between '(...)', because '<<' is an operator and therefore follows the precedence rules. In this case:

log_debug("VFD: Max frequency = " << (value / 100) << "Hz " << (value * 60 / 100) << "RPM");